src/python/pants/bin/remote_pants_runner.py

```diff
@@ -141,6 +141,7 @@ def _connect_and_execute(self, pantsd_handle):
     ng_env = NailgunProtocol.isatty_to_env(self._stdin, self._stdout, self._stderr)
     modified_env = combined_dict(self._env, ng_env)
     modified_env['PANTSD_RUNTRACKER_CLIENT_START_TIME'] = str(self._start_time)
+    modified_env['PANTSD_REQUEST_TIMEOUT_LIMIT'] = str(self._bootstrap_options.for_global_scope().pantsd_run_start_timeout)
```

Comment:
I think out of date?

Reply:
No, we actually need to pass the option as an environment variable, so that every run can have a different timeout. This is the best way I found to pass data from the client to the server.

Reply:
I missed that, sorry. You're right.